packages/amplify-graphql-model-transformer/package.json

```diff
@@ -21,7 +21,7 @@
     "access": "public"
   },
   "scripts": {
-    "build": "tsc && cd rds-lambda && tsc && bestzip --force node ../lib/streaming-lambda.zip ./lib*",
+    "build": "tsc && cd rds-lambda && npm install && tsc && cd lib && mkdir -p ./node_modules && cp -R ../node_modules/* ./node_modules/ && bestzip --force node ../../lib/rds-lambda.zip ./*",
```

Comment:
Should we delete the node_modules folder before running `npm install`?

Reply:
we could, I wasn't as worried about it since the CI runs from a fresh environment but I can add that in quick